src/iso19111/io.cpp

```diff
@@ -5352,13 +5354,17 @@ PROJStringSyntaxParser(const std::string &projString, std::vector<Step> &steps,
             } else if (word != "step") {
                 const auto pos = word.find('=');
                 auto key = word.substr(0, pos);
-                auto pair = (pos != std::string::npos)
-                                ? Step::KeyValue(key, word.substr(pos + 1))
-                                : Step::KeyValue(key);
-                if (steps.empty()) {
-                    globalParamValues.push_back(pair);
-                } else {
-                    steps.back().paramValues.push_back(pair);
+                if (!(dropEarlyBindingsTerms &&
```

Comment:
I wonder, With the changes you have pushed in c048292f3b8e408e6a81700a74d9b44d532227ac, is it really necessary to give `+towgs84`, `+nadgrids`, `+geoidgrids`, `+wktext` special treatment? If it is, I guess `+datum` should be added to the list as well.

Reply:
`+towgs84`, `+nadgrids`, `+geoidgrids` should at least receive special treatment since they might be stored in the preserved PROJ string in case there is a parameter that prevents building a full ISO19111 compliant CRS (we must preserve them when exporting the CRS as a PROJ string,but not when integrating it in a pipeline)
Regarding `datum`: if we drop it, then we must substitute it by its `ellps`. I'll try to do that.
+wktext dropping should indeed no longer be neeed

Reply:
Sure, makes sense. Thanks for the clarification.